CMakeLists.txt

```diff
@@ -4,7 +4,6 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/CMake)
 find_package(CUnit REQUIRED)
 find_package(FreeImage REQUIRED)
 find_package(GLEW REQUIRED)
-find_package(IniParser REQUIRED)
```

Comment:
RapidJSON is header only. Technically we probably should add that as a required package to ensure that the headers exist, but it's not needed for linking.